### PR TITLE
Error on creation of dbc file with float variables

### PR DESCRIPTION
**Pull request by gbmont**
Hi

If a signal of type float is defined, dbc.py (program lines 369 to 371)  raises an exception on creation of the an entry “SIG_VALTYPE_“ , probably because of wrong dictionary indexing.

Best regards
Geri

### CODE REVIEW

File: canmatrix/sym.py [hunk 1]
```diff
@@ -162,7 +162,10 @@ def sendreceive(f):
         for frame in frames:
             name = "[" + frame.name + "]\n"
 
-            idType = "ID=%08Xh" % (frame.id)
+            if frame.extended == 1:
```
Comment (altendky) [inline canmatrix/sym.py]:
```python
if frame.extended:
```

Comment (ebroecker) [reply canmatrix/sym.py]: I think it was this issue:


https://github.com/ebroecker/canmatrix/commit/cfec9b81ecbbc7ae0ab3ab1c2c1b756abffe01d5#diff-11bf3294736079e6a427e649315ff8f6

Comment (ebroecker) [reply canmatrix/sym.py]: Oh No

I Just cannot see this issues and Changes on my Phone ...
So forget my last comment

**Comment by altendky** [comment]
GitHub is messing this diff up.  @gbmont, your branch is out of date and the `.sym` change is already present in the `development` branch.

https://github.com/ebroecker/canmatrix/blame/395e45678eb41989ce2224461150bd682db13910/canmatrix/sym.py#L169-L172

Note how the branch is based off of b6583ad from October 9, 2017.
https://github.com/gbmont/canmatrix/commits/bugfix_version06

**Comment by gbmont** [comment]
Hello

Sorry for the confusion, I tried to create an issue and the report relates to the dbc-file.

Best regards
Geri

**Pull request closed without merging**